db/migrate/20201121134354_add_time_zone_offset_to_patients.rb

```diff
@@ -0,0 +1,12 @@
+class AddTimeZoneOffsetToPatients < ActiveRecord::Migration[6.0]
+  def up
+    add_column :patients, :time_zone_offset, :string
+    # -04:00 was being used before and we expect it to be updated
+    # to the proper value the next time each patient is saved.
+    Patient.update_all(time_zone_offset: '-04:00')
```

Comment:
Scope this query to purged false and update in batches.

Reply:
Not sure that -4:00 is the right choice because it really should only be -4:00 only if the address is not set. I think this migration should essentially be the same logic as the `address_timezone_offset` method.

Reply:
Since there is the before_save callback for setting the time_zone offset, then could that work just by iterating through patients in batches and saving? I originally did the update_all so that we just ran an SQL UPDATE command and then the time_zone offset would just be properly set the next time the patient was saved.

Reply:
`update_all` will not run the callbacks and in the likely case that a patient is not updated between migration and reminder time they would have the wrong time.